js/src/index.js

```diff
@@ -0,0 +1,266 @@
+import "main.css";
+import env from "../env.json";
+import bunzz from "bunzz-sdk";
+
+const DAPP_ID = env.JS_DAPP_ID;
+const API_KEY = env.JS_API_KEY;
+const NFT_STORAGE_KEY = env.JS_APP_NFT_STORAGE_KEY;
+let handler = null,
+  userAddress = null,
+  contract = null,
+  type = null,
+  blob = null,
+  base64 = null,
+  tokenId = null,
+  name = "",
+  description = "";
```

Comment:
hmmm... maybe like that?
```suggestion
let handler = null;
let userAddress = null;
let contract = null;
...
```